Block/Checkout/Success.php

```diff
@@ -151,14 +161,55 @@ public function getMultibancoData()
         return $result;
     }
 
+    /**
+     * If PresentToShopper resultCode and action has provided render this with the checkout component on the success page
+     * @return bool
+     */
+    public function renderAction()
+    {
+        if (
+            !empty($this->getOrder()->getPayment()->getAdditionalInformation('resultCode')) &&
```

Comment:
Is this `!empty` check even needed if you're comparing to `PresentToShopper` later?

Reply:
there is a potential that the resultCode does not exists, so that is why I added this check to prevent warnings.